src/pks/surface_balance/constitutive_relations/land_cover/seb_physics_funcs.cc

```diff
@@ -280,9 +279,11 @@ UpdateEnergyBalanceWithSnow_Inner(const GroundProperties& surf,
 
   // latent heat
   double vapor_pressure_skin = SaturatedVaporPressure(snow.temp);
-
+  double u_star = met.Us * c_von_Karman / std::log(met.Z_Us / CalcRoughnessFactor(snow.height, surf.roughness, snow.roughness));
+  double Re0 = params.density_air * u_star * CalcRoughnessFactor(snow.height, surf.roughness, snow.roughness) / params.dynamic_viscosity_air;
+  double KB = params.Da0_a * std::pow(Re0, params.Da0_b) - (params.Cd0_c * std::log(Re0) + params.Cd0_d);
```

Comment:
Can you update the documentation and/or comments in this file with a reference as to where these equations are coming from?

Reply:
Can we just remove these changes to air resistance? I am still hesitating to add this to the master branch.